templates/go/events/manifest.yaml

```diff
@@ -0,0 +1,12 @@
+builders:
+  default: paketobuildpacks/builder:base
+  base: paketobuildpacks/builder:base
+  full: paketobuildpacks/builder:full
+
+buildpacks:
+  - paketo-buildpacks/go-dist
+  - docker.io/lanceball/boson-go # github.com/lance/buildpacks
```

Comment:
Before this PR lands, we should figure out where these should really live - both git org/repo, and container repo.

/hold

Reply:
It seems to me at least that git's boson-project is the right place for the code, and docker.io is still the most common public registry.